WebJobs.Extensions.sln

```diff
@@ -40,65 +40,123 @@ EndProject
 		{7719584E-9863-405A-9D26-2CC02AFC2427}.Release|Any CPU.Build.0 = Release|Any CPU
+		{7719584E-9863-405A-9D26-2CC02AFC2427}.Release-Preview|Any CPU.ActiveCfg = Release|Any CPU
+		{7719584E-9863-405A-9D26-2CC02AFC2427}.Release-Preview|Any CPU.Build.0 = Release|Any CPU
 		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Debug-Preview|Any CPU.ActiveCfg = Debug|Any CPU
+		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Debug-Preview|Any CPU.Build.0 = Debug|Any CPU
 		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Release|Any CPU.Build.0 = Release|Any CPU
+		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Release-Preview|Any CPU.ActiveCfg = Release|Any CPU
+		{47B3ECC3-DDDB-4C80-8A13-E6279A2EB835}.Release-Preview|Any CPU.Build.0 = Release|Any CPU
 		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Debug-Preview|Any CPU.ActiveCfg = Debug|Any CPU
+		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Debug-Preview|Any CPU.Build.0 = Debug|Any CPU
 		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Release-Preview|Any CPU.ActiveCfg = Release|Any CPU
+		{4B1CA665-5F3E-4A0A-BEFC-FC174BB1BABA}.Release-Preview|Any CPU.Build.0 = Release|Any CPU
 		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Debug-Preview|Any CPU.ActiveCfg = Debug|Any CPU
+		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Debug-Preview|Any CPU.Build.0 = Debug|Any CPU
 		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Release-Preview|Any CPU.ActiveCfg = Release|Any CPU
+		{4F335FB9-2BE8-4B51-9CE9-01465CCF05CB}.Release-Preview|Any CPU.Build.0 = Release|Any CPU
 		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Debug-Preview|Any CPU.ActiveCfg = Debug-Preview|Any CPU
+		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Debug-Preview|Any CPU.Build.0 = Debug-Preview|Any CPU
 		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Release|Any CPU.Build.0 = Release|Any CPU
+		{A0DE3BF4-AD56-44D9-8404-8E0778AE5CA0}.Release-Preview|Any CPU.ActiveCfg = Release-Preview|Any CPU
```

Comment:
Solution files are always hard to understand, but the gist is:

1. I added 2 new configurations, `Debug-Preview` and `Release-Preview`.
2. When that config is active in Visual Studio, only specific projects (the ones with a preview) actually 'use' it. All others just map back to 'debug' or 'release'.

End result is, you select `debug-preview` in VS and only CosmosDB and CosmosDB.Tests will actually be `debug-preview`, everything else is still just `debug`.